modules/cardano-service.nix

```diff
@@ -49,6 +48,7 @@ let
     "--configuration-key ${config.deployment.arguments.configurationKey}"
     "--topology ${cfg.topologyYaml}"
     "--node-id ${params.name}"
+    ("--policies " + (if ("c" == builtins.substring 0 1 "${params.name}") then "${./../policy_core.yaml}" else "${./../policy_relay.yaml}"))
```

Comment:
the node type may also be able to set this correctly

Reply:
https://github.com/input-output-hk/iohk-ops/blob/50df339b40b5f75d796eb2e22ead02cbad4cd5c5/modules/cardano-service.nix#L51

Is the above ok?